googletest/CMakeLists.txt

```diff
@@ -93,6 +93,14 @@ target_link_libraries(gtest_main gtest)
 
 ########################################################################
 #
+# Install rules
```

Comment:
gtest and gtest_main libraries are installed to `<CMAKE_INSTALL_PREFIX>/lib`

The `include/gtest` directory is installed to `<CMAKE_INSTALL_PREFIX>/include`